src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    partnerName,
+    sex,
+    isMarried,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age && (
+        <p className="Person__age">{`I am ${age}`}</p>
+      )
+      }
+
+      {!isMarried ? (
+        <p className="Person__partner">I am not married</p>
+      ) : (
+        <p className="Person__partner">
+          {`${partnerName} is my ${
+            sex === 'm' ? 'wife' : 'husband'
+          }`}
```

Comment:
Please create a variable for this condition, as Olya indicated in the previous review. It’s worth getting used to doing this in order for your code to be cleaner, since in the future there will be much more such conditions